src/System.Runtime.InteropServices/src/System/Runtime/InteropServices/RuntimeEnvironment.cs

```diff
@@ -23,7 +23,10 @@ public static bool FromGlobalAccessCache(System.Reflection.Assembly a)
         }
         public static string GetRuntimeDirectory()
         {
-            return Path.GetDirectoryName(typeof(object).Assembly.Location) + Path.DirectorySeparatorChar;
```

Comment:
=> In CoreFX this is wherever System.Private.Corelib is
        That's bcoz GetRuntimeDirectory() use typeof(object).Assembly and object is in CoreLIB. But 
        msdn says "Returns the directory where the common language runtime is installed". That's 
       why i doubt if the current CoreFX implementation is wrong. Now now i will leave it as it is.